RoddyCore/test/src/de/dkfz/roddy/config/validation/DefaultValidatorSpec.groovy

```diff
@@ -37,17 +52,46 @@ class DefaultValidatorSpec extends Specification {
         ConfigurationValue cval = new ConfigurationValue('bla', value, type)
 
         then:
-        cval.valid == expected
         cval.warnings == expectedWarnings
-        cval.errors == []
+        cval.errors == expectedErrors
+        cval.valid == expected
 
         where:
         value                   | type     | expected | expectedWarnings                          | expectedErrors
+        // All are ok
         'abc${var}bcd${var}'    | 'string' | true     | []                                        | []
         'abc${var}bcd${var}def' | 'string' | true     | []                                        | []
-        'abc$'                  | 'string' | false    | [unattachedDollarCharacter.expand('bla')] | []
-        'abc${'                 | 'string' | false    | [unattachedDollarCharacter.expand('bla')] | []
-        'tr$ue'                 | 'string' | false    | [unattachedDollarCharacter.expand('bla')] | []
 
+        '\\$'                   | 'string' | true     | []                                      | []  //Escaped dollar
+        'abc\\$'                | 'string' | true     | []                                      | []  //Escaped dollar
+        'abc\\${'               | 'string' | true     | []                                      | []  //Escaped dollar with open brace
+        'a\\${b}c'              | 'string' | true     | []                                      | []  //Escaped dollar with complete braces
+
+        // Not ok
```

Comment:
If the ConfigurationIssue defines a detachedDollarCharacter with the following warning

> "Variable '#REPLACE_0#' contains plain dollar sign(s) without braces. Roddy does not interpret them as variables and cannot guarantee correct ordering of assignments for such variables in the job parameter file."

I would expect that a test for 
`'abc$'                  | 'string' | false    | [detachedDollarCharacter.expand('bla')] | []`
yields the same result as
`'$abc'                  | 'string' | false    | [detachedDollarCharacter.expand('bla')] | []`

however `$abc` throws no `detachedDollarCharacter` warning.

Reply:
At least now we have 'tr$ue' throws. So likely '$abc' throws either. To be 100% certain a dedicated '$true' could be added, but I would not expect much difference from 'tr$ue'.